variables.tf

```diff
@@ -1,10 +1,11 @@
 variable "logs_cloudwatch_retention" {
   description = "Specifies the number of days you want to retain log events in the log group."
-  default     = 90
-  type        = "string"
+  #  default     = 90
```

Comment:
Why comment out the default?

Reply:
This was cruft left over from me trying to get it to work initially. Sorry!